### ISSUE TITLE
Missing Dependency in generated Java output

### ISSUE THREAD

**Issue by GerrietReents** [btc-ag/service-idl#118]
For the following idl
```
 module test {
 	
 	struct A { 
 		int64 a;
 	}; 
 	
 	struct B {  
 		int64 b; 
 	};  
 	
 	interface AAcc [ guid = A50296FB-8BE6-4E78-A71A-76F70E927C91 ]
	{
 		query getA(in sequence<uuid> ids) returns sequence<A>;
 	};
 	
 	interface BAcc [ guid = F3B03063-F8ED-4D88-AFC6-A11DC3E63D82 ] : AAcc {
 		query getB(in sequence<uuid> ids) returns sequence<B>;
 	};
 	
 }
```
the  pom.xml for com.test.bacc is missing a dependency to com.test.aacc.

**Comment by sigiesec** [comment]
This will probably be fixed in the context of #42

**Comment by sigiesec** [comment]
With the current version, I cannot reproduce this anymore. Can you please check this on your side? Please reopen the issue if it persists.
